modules/apps/29-fee/ibc_middleware_test.go

```diff
@@ -815,7 +852,11 @@ func (suite *FeeTestSuite) TestOnTimeoutPacket() {
 			tc.malleate()
 
 			err = cbs.OnTimeoutPacket(suite.chainA.GetContext(), packet, relayerAddr)
-			suite.Require().NoError(err)
+			if tc.expLocked {
+				suite.Require().False(suite.chainA.GetSimApp().IBCFeeKeeper.IsLocked(suite.chainA.GetContext()))
+			} else {
+				suite.Require().True(suite.chainA.GetSimApp().IBCFeeKeeper.IsLocked(suite.chainA.GetContext()))
+			}
```

Comment:
I'd swap these. If tc.expLocked == true, I'd expect `IsLocked` to also return true. exp == expected